src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -66,9 +66,19 @@ If `instrument.js` displays in your console while debugging, add Sentry to your
 
 ## Dealing with Ad-Blockers
 
+When you are using our CDN, ad-blocking or script-blocking extensions may prevent our SDK from being fetched and initialized properly. Because of this, any call to SDKs API will fail and may cause your application to misbehave.
```

Comment:
```suggestion
When you are using our CDN, ad-blocking or script-blocking extensions may prevent our SDK from being fetched and initialized properly. Because of this, any call to the SDKs API will fail and may cause your application to behave unexpectedly.
```